apis/k8ssandra/v1alpha1/constants.go

```diff
@@ -58,6 +58,11 @@ const (
 	K8ssandraClusterNameLabel      = "k8ssandra.io/cluster-name"
 	K8ssandraClusterNamespaceLabel = "k8ssandra.io/cluster-namespace"
 
+	// TODO this is a workaround because cass-operator does not let us set k8ssandra.io/* labels in
+	//  AdditionalServiceConfig. Remove and replace by the two original constants once this is fixed.
+	TmpK8ssandraClusterNameLabel      = "not.k8ssandra.io/cluster-name"
+	TmpK8ssandraClusterNamespaceLabel = "not.k8ssandra.io/cluster-namespace"
```

Comment:
This requires a fix in cass-operator.
I will mark the PR ready for review once that is done and we can revert to the original labels.

Reply:
k8ssandra/cass-operator#689